File: 코딩 기초 트레이닝/A 강조하기-p181874/A 강조하기.js
// https://school.programmers.co.kr/learn/courses/30/lessons/181874

function solution(myString) {
    myString = myString.split("");
    myString = myString.map((d) => {
        if(d === 'a' || d === 'A') {
            return 'A';
        } else {
            return d.toLowerCase();
        }
    });

    return myString.join('');
}

File: 코딩 기초 트레이닝/A 강조하기-p181874/A 강조하기.py
# https://school.programmers.co.kr/learn/courses/30/lessons/181874

def solution(myString):
    answer = ''
    myString = list(myString)

    for m in myString:
        if m == 'a' or m == 'A':
            answer += 'A'
        else:
            answer += m.lower()

    return answer
